apps/web/src/client/components/accessible-autocomplete.js

```diff
@@ -0,0 +1,12 @@
+import 'accessible-autocomplete';
```

Comment:
I would change this to a dynamic import and only loaded if the elements you are looking bellow exists on the page.

Something like this

```js
const accessibleAutocompleteEls = document.querySelectorAll('select.accessible-autocomplete');

if (accessibleAutocompleteEls.length > 0) {
  import('accessible-autocomplete').then(() => {
    [...accessibleAutocompleteEls].forEach((select) => {})
  });
}
```

Reply:
I see what you mean, but isn't the app.js just a single bundle of all the js in the app? A conditional import would only be worthwhile if this import can be fetched lazily? Or is that what happens?

Reply:
That is what it will happen, a different chunk will be split and the code will fetch it only if your IF executes.